privaterelay/locales/en-us/privaterelay/app.ftl

```diff
@@ -0,0 +1,226 @@
+#   $attrs (string) - specific attributes added to external links
+faq-question-4-answer-html = { -brand-name-relay } does not yet offer the ability to reply using an alias. 
+    If you try, nothing will happen. We are planning an additional feature to let you <a href="{ $url }" {$attrs}>reply anonymously to senders</a>.
+faq-question-5-question = Can I make up my own { -brand-name-relay } alias using the @relay.firefox.com domain?
+faq-question-5-answer = Not currently, but we are considering new features including letting you create your own alias with a designated domain.
+faq-question-6-question = What happens if { -brand-name-mozilla } shuts down the { -brand-name-firefox-relay } service?
+faq-question-6-answer = We will give you advance notice that you need to change the email address of any accounts that are using { -brand-name-relay } aliases.
+faq-question-7-question = What if an email sent to my alias contains an attachment?
+faq-question-7-answer = We now support attachment forwarding. However, there is a 150KB limit for email forwarding using { -brand-name-relay }. 
+    Any emails larger than 150KB will not be forwarded.
+
+## Profile Page (Logged In) - https://dev.fxprivaterelay.nonprod.cloudops.mozgcp.net/accounts/profile/
+
+# Variables:
+#   $email (string) - User email address
+profile-label-welcome = Welcome, { $email }!
+profile-headline-manage-relay = Manage your { -brand-name-relay } aliases
+profile-headline-manage-domain = Manage your Domain aliases
+profile-supports-email-forwarding = { -brand-name-firefox-relay } supports email forwarding (including attachments) of email up to 150KB in size
+profile-promo-premium-headline = Go premium for more features.
+profile-promo-premium-copy = With { -brand-name-relay } Premium, you get your own email domain.
+profile-promo-premium-cta = Buy Premium
+profile-label-saved = Label saved!
+profile-label-generate-new-alias = Generate New Alias
+profile-label-delete = Delete
+
+# This string is followed by an email address
+profile-label-forward-emails = Forward emails to:
+
+# This string is followed by date
+profile-label-first-emailed = First emailed:
+
+# This string is followed by date:
+profile-label-created = Created:
+profile-label-details-show = Show Details
+profile-label-details-hide = Hide Details
+profile-label-forwarding = forwarding
+profile-label-blocking = blocking
+profile-label-copied = Copied!
+profile-label-blocked = Blocked
+profile-label-forwarded = Forwarded
+profile-label-cancel = Cancel
+profile-blocked-copy = { -brand-name-firefox-relay } will delete messages before they can reach your inbox when you select blocking for this alias.
+profile-forwarded-copy-html = { -brand-name-firefox-relay } will send messages to your inbox when you select forwarding for this alias. <br />
+    <span class="fwd-note"><span class="bold ff-Met">Note:</span> Email (including attachments) larger than 150KB are not currently supported and will not be forwarded.
```

Comment:
Can you please confirm these HTML tags?
I see two opening `<span` tags but only one closing `</span>` tag.
My guts are saying this should be a combined `<span class="fwd-note bold ff-Met">` tag.

Reply:
Can we do away the CSS code in the string? If not, yeah, if we can consolidate them, that's better/cleaner. so it won't risk being broken up.

Reply:
I updated this entire string structure. It's now a standard `<strong>` tag, and no HTML is included in the string.